crates/apps/src/lib/cli.rs

```diff
@@ -5601,6 +5630,45 @@ pub mod args {
         }
     }
 
+    impl Args for ShieldedSync<CliTypes> {
+        fn parse(matches: &ArgMatches) -> Self {
+            let ledger_address = LEDGER_ADDRESS_DEFAULT.parse(matches);
+            let last_query_height = BLOCK_HEIGHT_OPT.parse(matches);
+            let viewing_keys = VIEWING_KEYS.parse(matches);
```

Comment:
do we not want to support passing in spending keys here?

Reply:
Whenever a spending key is added to the wallet, its corresponding viewing key is also added under the same name. So I'm thinking that that this code should be sufficient for now. No?

Reply:
Sure, I just wondered if we considered it important.